### ISSUE TITLE
"Error [ERR_PACKAGE_PATH_NOT_EXPORTED]" after upgrade to Node 12.17.0

### ISSUE THREAD

**Issue by dshenderson** [ericelliott/riteway#291]
Basic installation as described in README. `package.json` has:

```
  "scripts": {
    "test": "nyc riteway -r esm test/**/*.test.js | tap-nirvana",
  },
  "devDependencies": {
    "esm": "^3.2.25",
    "nyc": "^15.1.0",
    "riteway": "^6.2.0",
    "tap-nirvana": "^1.1.0",
  }
```

Works perfectly in Node versions up to 12.16.3. With Node versions 12.17.0 and more recent, the following error is thrown:

> \> riteway -r esm test/**/*.test.js
> 
> internal/modules/cjs/loader.js:491
>   throw new ERR_PACKAGE_PATH_NOT_EXPORTED(basePath, mappingKey);
>   ^
> 
> Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './bin/tape' is not defined by "exports" in /path/to/modules/node_modules/tape/package.json
>     at applyExports (internal/modules/cjs/loader.js:491:9)
>     at resolveExports (internal/modules/cjs/loader.js:507:23)
>     at Function.Module._findPath (internal/modules/cjs/loader.js:635:31)
>     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:953:27)
>     at Function.Module._load (internal/modules/cjs/loader.js:842:27)
>     at Module.require (internal/modules/cjs/loader.js:1026:19)
>     at require (internal/modules/cjs/helpers.js:72:18)
>     at Object.<anonymous> (/path/to/modules/node_modules/riteway/bin/riteway:3:1)
>     at Module._compile (internal/modules/cjs/loader.js:1138:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10) {
>   code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
> }

**Comment by redbar0n** [comment]
I second this. I don't think it has to do with the Node version, as I am on a newer version than those.

See this: https://github.com/substack/tape/pull/530

> "the tape binary should only be executed, never required."

This seems to be the cause, @ericelliott: https://github.com/ericelliott/riteway/blob/e647631c3abb4b05e1edb2bd69dde4d282072eef/bin/riteway#L3

**Comment by redbar0n** [comment]
Confirmed that the fix in that commit worked to monkey-patch `tape/package.json` : https://github.com/substack/tape/pull/530/commits/cc380d59e33a794cbb18b2dc23da2d1c077e8c97

**Comment by ericelliott** [comment]
Should be fixed in riteway@6.2.1 -- reopen if it isn't.

**Comment by dshenderson** [comment]
Thank you all for resolving this so quickly!
